m3u.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/grafov/m3u8"
```

Comment:
**[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 could not import github.com/grafov/m3u8 (cannot find package "github.com/grafov/m3u8" in any of: